docs/Project-Configuration/Project-Commands.md

```diff
@@ -397,18 +397,38 @@ Any updates to the expansions made with this command will only persist for the d
   params:
     ignore_missing_file: true
     file: src/ec2_artifacts.yml
+    redact_file_expansions: true
 
 - command: expansions.update
   params:
     updates:
     - key: artifact_url
       value: http://s3/static-artifacts.tgz
+
+- command: expansions.update
+  params:
+    updates:
+    - key: artifact_url
+      concat: tgz
+
+- command: expansions.update
+  params:
+    updates:
+    - key: artifact_url
+      value: http://s3/static-artifacts.tgz
+      redact: true
 ```
 
 Parameters:
 
--   `updates`: key-value pairs for updating the task's parameters
+-   `updates`: a list of expansions to update.
+        - `key`: the expansion key to update. (required)
+        - `value`: the new value for the expansion.
+        - `concat`: the string to concatenate to the existing value. Per
+           update, only `value or `concat` can be set.
+        - `redact`: if true, the expansion will be redacted in the task logs.
 -   `file`: filename for a YAML file containing expansion updates
+-   `redact_file_expansions`: if true, the expansions added from the file will be redacted in the task logs
```

Comment:
If a user specifies something weird like `redact: false` for a single expansion + `redact_file_expansions: true`, which one takes precedence? Also might be weird if people try combinations like `redact: true` + `redact_file_expansions: false`. Can we clarify how they interact when they conflict?

Reply:
Writing docs is hard, I went through a couple iterations lmk what you think!

Reply:
I know, writing clear and unambiguous English is so difficult as an engineer... I think it overall reads pretty well though! Being explicit about `redact: false` as I mentioned in the other comment would be the only improvement I could suggest.